### ISSUE TITLE
Old OkHttp version causes crash on Android 11 devices

### ISSUE THREAD

**Issue by VladBytsyuk** [DataDog/dd-sdk-android#658]
Hello DataDog team!

When I'm trying to run my app that uses `1.9.0` version on Android 11
```
    implementation group: 'com.datadoghq', name: 'dd-sdk-android', version: '1.9.0'
    implementation group: 'com.datadoghq', name: 'dd-sdk-android-ktx', version: '1.9.0'
```
calling `com.datadog.android.log.Logger.Builder()` causes error:
```
java.lang.ExceptionInInitializerError
        at okhttp3.internal.platform.Platform.get(Platform.java:85)
        at okhttp3.OkHttpClient.newSslSocketFactory(OkHttpClient.java:263)
        at okhttp3.OkHttpClient.<init>(OkHttpClient.java:229)
        at okhttp3.OkHttpClient$Builder.build(OkHttpClient.java:1015)
        at com.datadog.android.core.internal.CoreFeature.<clinit>(CoreFeature.kt:89)
        at com.datadog.android.log.Logger$Builder.<init>(Logger.kt:181)
        ...
```

It looks like a known issue of OkHttp version `3.12.6`.
Link: https://github.com/square/okhttp/issues/5816

**Describe what you expected:**
Please update the OkHttp version.

**Additional context**
 - Android OS version: 11
 - Datadog SDK version: 1.9.0
 - Versions of any other relevant dependencies:  OkHttp: 3.12.6

**Comment by 0xnm** [comment]
Hello @VladBytsyuk! Thanks for the reporting the issue. Can you please help us to assess the criticality? Do you see this crash on all Android 11 devices or just on some? If the latter, what percent of devices is affected?

**Comment by VladBytsyuk** [comment]
I've seen this crash on all my devices (Samsung SM-T510, OnePlus GM1910, Emulator). 
It isn't production code by now and I can't five more info.

**Comment by 0xnm** [comment]
@VladBytsyuk thanks for coming back to us. This issue will be fixed in the next `1.10.0` release. It happens because OkHttp version gets resolved to the one used by DataDog Android SDK after dependency graph evaluation (you can check graph by following the steps described [here](https://docs.gradle.org/current/userguide/viewing_debugging_dependencies.html)), but if it is blocking issue for you, this can be solved on your side in the meantime as well: you can force resolver to use the newer version of OkHttp by adding explicit `implementation(okhttp version)` block to the `build.gradle` or by using [override mechanism](https://docs.gradle.org/current/userguide/dependency_downgrade_and_exclude.html).

**Comment by VladBytsyuk** [comment]
Good! Thank you!

**Comment by 0xnm** [comment]
Fix is merged and will be available in the next release of `1.10.0` branch.
